__tests__/integration.test.js

```diff
@@ -14,36 +14,67 @@ describe('selectTopics()', () => {
       .get('/api/topics')
       .expect(200)
       .then(({ body }) => {
-          expect(body.topics).toHaveLength(3);
-          expect(body.topics[0]).toMatchObject({
-            slug: 'mitch',
-            description: 'The man, the Mitch, the legend',
-          });
+        expect(body.topics).toHaveLength(3);
+        expect(body.topics[0]).toMatchObject({
+          slug: 'mitch',
+          description: 'The man, the Mitch, the legend',
+        });
+      });
+  });
+});
+describe('error handlingTopics()', () => {
+  test('returns an error with invalid endpoint', () => {
+    return request(app)
+      .get('/api/topic')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not Found');
+      });
+  });
+});
+describe('getApis()', () => {
+  test('return Json object with all apis endpoint in file ', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.dataApi).toEqual(jsonFile);
       });
   });
-    
 });
 
-describe('error handling()', () => {
-    test('returns an error with invalid endpoint', () => {
-      return request(app)
-        .get('/api/topic')
-        .expect(404)
-        .then(({ body }) => {
-            expect(body.msg).toBe('Not Found');
+describe('/api/articles/:article_id', () => {
+  test('respond with a 200 status code with an individual article object ', () => {
+    return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
```

Comment:
if the test data gets updated at any point then this test will fail - how can you refactor this test so that it doesn't include the actual article data in it but still test for the same thing?